sw/device/silicon_creator/imm_rom_ext/prebuilts/nop.S

```diff
@@ -0,0 +1,8 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+.section .rom_ext_immutable, "ax"
+.globl _start
+_start:
```

Comment:
```suggestion
.globl _imm_rom_ext_start
_imm_rom_ext_start:
```

Let's make this symbol more specific since it is global.

Reply:
Done.